app-admin/rcm/rcm-1.3.1.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="rc file (dotfile) management"
+HOMEPAGE="https://github.com/thoughtbot/rcm"
+SRC_URI="https://thoughtbot.github.io/rcm/dist/rcm-1.3.1.tar.gz"
```

Comment:
We can use some [variables](https://devmanual.gentoo.org/ebuild-writing/variables/) to make future bumps easier, eg: ``SRC_URI="https://thoughtbot.github.io/${PN}/dist/${P}.tar.gz"``